src/runtime/Object.h

```diff
@@ -150,6 +150,15 @@ class ObjectPropertyName {
         }
     }
 
+    Value toPropertKeyString() const
+    {
+        if (isUIntType()) {
+            return Value(String::fromDouble(uintValue()));
```

Comment:
This new function looks great. Each PropertyKey has one of two types which are string or symbol. So, converted number should be restored to string type. BTW how about changing the name of this function to `toPropertyKeyValue` since PropertyKey could be a symbol and the return type is Value also.

Reply:
The name is up to changes of course, i just came up with something on the spot. Gonna update it soon with your suggestion.